app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= stylesheet_link_tag "recipes", "data-turbo-track": "reload" %>
+
+<h2>Recipes</h2>
+
+<div class="add-food-btn">
+  <%= link_to 'Add Recipe', new_recipe_path, class: 'btn btn-primary' %>
+</div>
```

Comment:
[REQUIRED] Kindly remove this button as it is not in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/recipe_index.png?token=GHSAT0AAAAAACHZNTDQE2VEKSTBM5DQEN3GZJHHSRQ)